### PR TITLE
fix: cli flickering

### PR DESCRIPTION
**Pull request by anchenyi**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/27092892

### CODE REVIEW

**Comment by anchenyi** [comment]
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/27092892

File: packages/cli/package.json [hunk 1]
```diff
@@ -113,6 +114,7 @@
     "express": "^4.18.2",
     "figures": "^3.2.0",
     "fs-extra": "^9.1.0",
+    "inquirer": "^7.3.3",
```
Comment (jayzhang) [inline packages/cli/package.json:117]: new version of inquirer is 9.2.x, can we use new version?

File: packages/cli/src/userInteraction.ts [hunk 2]
```diff
@@ -431,14 +435,16 @@ class CLIUserInteraction implements UserInteraction {
     if (config.validation || config.additionalValidationOnAccept) {
       validationFunc = async (input: string) => {
         let res: string | undefined = undefined;
+        const spinner = new CustomizedSpinner();
+        spinner.start();
```
Comment (jayzhang) [inline packages/cli/src/userInteraction.ts:439]: Make sure the UX is OK in terminals with different backgroud color: especially for white and black.

**Review by jayzhang — DISMISSED** [review]

File: packages/cli/package.json (hunk 1 above)
Comment (anchenyi) [reply packages/cli/package.json:117]: New version causes the flicks, so we need to downgrade the version to 7.x to fix this issue.

File: packages/cli/src/userInteraction.ts (hunk 2 above)
Comment (anchenyi) [reply packages/cli/src/userInteraction.ts:439]: The UX works for white, black, and blue background color

**Review by jayzhang — APPROVED** [review]

**Pull request merged**